templates/components/products/price.html

```diff
@@ -1,16 +1,20 @@
+<!--products have options-->
+{{#if product.options.length}}
```

Comment:
`has_options` is always undefined here because it's not available on the product model (https://stencil.bigcommerce.com/docs/product-resources).  instead it's on the product card model (https://stencil.bigcommerce.com/docs/common-product-card-model)